app/controllers/state_file/questions/eligibility_offboarding_controller.rb

```diff
@@ -5,12 +5,15 @@ class EligibilityOffboardingController < QuestionsController
       before_action :set_prev_path, only: [:edit]
       helper_method :ineligible_reason
 
+      before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
+
       def ineligible_reason
         key = current_intake.disqualifying_eligibility_answer
         if key.present?
           I18n.t(
             "state_file.questions.eligibility_offboarding.edit.ineligible_reason.#{key}",
-            state: current_state_name
+            state: current_state_name,
+            filing_year: @filing_year
```

Comment:
Want to flag this change for CFA, I noticed all the other eligibility offboarding reasons have years hardcoded in the `en.yml` and `es.yml` files.

Didn't want to presume that I should change all those as well to use `filing_year` instead, but wanted to flag in case you all did want to take that on.

Reply:
yes, that's being addressed, thank you for the flag!